Keas.Mvc/Controllers/Api/EquipmentController.cs

```diff
@@ -64,12 +64,17 @@ public async Task<IActionResult> GetEquipmentInSpace(int spaceId)
 
         public async Task<IActionResult> CommonAttributeKeys() 
         {
-            var keys = await _context.EquipmentAttributes
-            .Where(x => x.Equipment.Team.Slug == Team)
-            .GroupBy(x => x.Key)
-            .Take(5)
-            .OrderByDescending(x => x.Count())
-            .Select(x => x.Key).AsNoTracking().ToListAsync();
+            //var keys = await _context.EquipmentAttributes
+            //.Where(x => x.Equipment.Team.Slug == Team)
+            //.GroupBy(x => x.Key)
+            //.Take(5)
+            //.OrderByDescending(x => x.Count())
+            //.Select(x => x.Key).AsNoTracking().ToListAsync();
+
+            var keys = await _context.EquipmentAttributeKeys
+                .Where(a => a.TeamId == null || a.Team.Slug == Team)
+                .OrderBy(a => a.TeamId).ThenBy(a => a.Key)
```

Comment:
@jSylvestre I think we should only orderBy `key`.  Then we can index on key and the whole thing will be fast, and I don't think we care which ones are global and which are not, at least from this query